.github/workflows/test-action.yml

```diff
@@ -0,0 +1,19 @@
+name: Test main.py
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      docker:
+        image: docker:19.03.12
+        options: --privileged
+    steps:
+    - uses: actions/checkout@v4
+    - name: Run main.py with fake model
+      run: |
+        export SANDBOX_CONTAINER_IMAGE="ubuntu:24.04"
+        python -m pip install -r requirements.txt
+        PYTHONPATH=`pwd` python ./opendevin/main.py -d ./ -t "write a hello world script" --model-name=fake
```

Comment:
@xingyaoww would be good to run this a second time with the codeact agent (which I broke in an earlier PR 🙃). Would it be hard to respect the `fake` model and just return a no-op?

Reply:
Sounds good, will look at this when i finished #105